### DISCUSSION TITLE
Please add a DM message system.

### DISCUSSION THREAD

**Discussion by happy-spark** [community/community#34049] [Other Feature Feedback, Questions, & Ideas] [answered]
Please add a DM message system. I think DM system is essential for collaboration among developers.

**Comment by Sagar-Sharma-7** [comment]
Yes, I agree.
It's gonna be really helpful for all the developers to communicate and collaborate with other developers.

**Comment by YeahJenni** [comment]
YESSS!!!!!!!!!
PLEASEEE!!!
Will help very much with getting help from devs, privately

**Reply by happy-spark** [reply]
Thank you🙂

**Comment by raelsei** [comment]
Good idea 🔥

**Reply by happy-spark** [reply]
👍

**Comment by siddhpant** [comment, marked as answer]
No need for this, you can just show your email / IRC / Discord / social media ID on your profile.

Apparently GitHub used to have a DM feature, and they removed it.

GitHub must not become Instagram.

**Reply by PRIVACYisPLEASING** [reply]
Understandable that contact may be made through email yet depending on how an individuals mailbox is set up could pose issues; if one is not added as a contact could be sent to spam, also a link must be attached in email communications regarding why one is reaching out, listing email address could cause flooding as anyone can email, also it just makes more sense to integrate that function here as the communication would pertain to their prev, current, future work on github more than likely not span outside the scope. To me it feels strange to send out an email if I am not currently or never have worked on a project with an individual before and am not intending too but quite simply would like to ask a question as email is very personal to many and if it isn't personal its spam or advertisements. I feel it tobe unfair to those that wish to make their contact details available and unfair to those that wish to reach out. So much is done here its as if mountains are being moved. If this dm could be reconsidered, possibly provide a list of concerns regarding dm, so that the community could give feedback on how to mitigate those concerns within an x amount of time and if successful, then let it be implemented again. Just a suggestion/ request. LinkedIn is very professional & was able to not become like Instagram, so why can't github do the same? I didn't check but I'm sure individuals that helped write the code for LinkedIn as well Instagram are on here or at least the code is probably here. So far more agree to have DM then disagree. Maybe a vote could be allowed without any promises but just to have it be taken into consideration if the majority would very much like to have dm back? :-) :-) :-)

**Reply by siddhpant** [reply]
> To me it feels strange to send out an email if I am not currently or never have worked on a project with an individual before and am not intending too

Then you should feel strange to send unsolicited DMs too. Emails are public so its expected to contact there.

> LinkedIn is very professional & was able to not become like Instagram

Use LinkedIn more.
